### PR TITLE
Add 6.8.16 release notes

### PR DESCRIPTION
**Pull request by tiagomlalves**

### CODE REVIEW

File: DSE_6.8_Release_Notes.md [hunk 1]
```diff
@@ -1,6 +1,31 @@
 # Release notes for DataStax Enterprise 6.8
 DSE 6.8.x is compatible with Apache Cassandra&trade; 3.11 and adds additional production-certified changes, if any. Components that are indicated with an asterisk (&ast;) (if any) are known to be updated since the prior patch version. 
 
+# Release notes for 6.8.16
+
+7 October 2021
```
Comment (msmygit) [inline DSE_6.8_Release_Notes.md:6]: *"Component versions for DSE `6.8.16`"* sub-section is missing on here which will actually state what is the current version and will also have an astericks mark which would represent that version changed from the prior DSE `6.8.15` patch version. See [here](https://github.com/datastax/release-notes/blob/master/DSE_6.8_Release_Notes.md#components-versions-for-dse-6815) for a reference example.

File: DSE_6.8_Release_Notes.md [hunk 2]
```diff
@@ -1,6 +1,31 @@
...
+
+## 6.8.16 DSE Cassandra
+
+* The Change Data Capture (CDC) is now near real time as in Cassandra 4.x. Active commitlog segments can be processed from the CDC raw directory defined in your {{cassandra.yaml}} without flushing tables. It also allows you to read the last mutation form the most recent commitlog file. See the Datastax Change Data Capture documentation page for more details. (DSP-21992)
```
Comment (msmygit) [inline DSE_6.8_Release_Notes.md:10]: Will {{cassandra.yaml}} render properly? I _think_ it'd be better to do it as `cassandra.yaml` (surrounding with single quotes)

File: DSE_6.8_Release_Notes.md [hunk 3]
```diff
@@ -1,6 +1,31 @@
...
+
+## 6.8.16 DSE core
+
+* Added check against the negative value in local stream throughput (stream_throughput_outbound_megabits_per_sec) and inter dc stream throughput (inter_dc_stream_throughput_outbound_megabits_per_sec) (DB-5010)
```
Comment (msmygit) [inline DSE_6.8_Release_Notes.md:14]: Let's use single quotes to surround the property names so it appears properly formatted in the final version.

File: DSE_6.8_Release_Notes.md (hunk 2 above)
Comment (johnsmartco) [reply DSE_6.8_Release_Notes.md:10]: Tiago, thanks. If not a hassle, can you insert the documentation URL on L10?  

[documentation](https://docs.datastax.com/en/dse/6.8/dse-admin/datastax_enterprise/config/configCDCLogging.html)

The updated topic will be in place shortly after 6.8.16 GAs.

File: DSE_6.8_Release_Notes.md (hunk 1 above)
Comment (tiagomlalves) [reply DSE_6.8_Release_Notes.md:6]: good catch - my bad. fixed

**Pull request merged**